internal/environment/environment.go

```diff
@@ -17,23 +17,25 @@ import (
 )
 
 var (
-	VPNPortmin = 5000
-	VPNPortmax = 6000
+	VPNPortmin   = 5000
+	VPNPortmax   = 6000
+	BeginnerType = 0 // Starts all labs so they are ready for the event
+	AdvancedType = 1 // Only starts lab dependencies such as guac and VPN
 )
```

Comment:
```suggestion
)
type labType uint8

const (
	BeginnerLab labType = iota
	AdvancedLab
)
```
Consider doing like this as this makes it easy to add future lab types and to create a function for converting the type to a human readable string